<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogTableSelect</class>
 <widget class="QDialog" name="DialogTableSelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>390</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>301</width>
     <height>321</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>报表类型：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>计划报表</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>学生报表</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>学院报表</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>起始时间：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="startTime"/>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>排序方式：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="total">
        <item>
         <property name="text">
          <string>年度申报数</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>获得支持的项目数</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>资金总数</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>支持率</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>验收通过率</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="student">
        <item>
         <property name="text">
          <string>学号</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>姓名</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>项目数量</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>验收通过率</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="college">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>项目数量</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>验收通过率</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>验收优良率</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>未结题率</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>结束时间：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="endTime"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogTableSelect</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogTableSelect</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
